apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx

```diff
@@ -683,6 +685,13 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
             });
     };
 
+    /**
+     * Handle confirm QR code regeneration checkbox event
```

Comment:
Shall we add a full stop at the end of the sentence? Check other places also.

Reply:
Fixed with 8535eecb987fefadc2bc7058e3a9070e260baf10